aws/lambda/github-status-test/lambda_function.py

```diff
@@ -23,10 +23,13 @@ def download_log(full_name, conclusion, job_id):
     url = f"https://api.github.com/repos/{full_name}/actions/jobs/{job_id}/logs"
     headers = {
         "Accept": "application/vnd.github.v3+json",
-        "Authorization": f"token {GITHUB_TOKEN}",
     }
-    r = requests.get(url, headers=headers)
-    log_data = r.content
+    for token in GITHUB_TOKENS.split(","):
```

Comment:
If you want the tokens to be used evenly, you could do a `random.choice(range(len(GITHUB_TOKENS.split(","))))` to select the token and not worry about rate limit (as if we see it again meaning more accounts are needed).